lib/node_modules/@stdlib/blas/ext/base/sdsapxsumpw/lib/ndarray.js

```diff
@@ -21,7 +21,7 @@
 // MODULES //
 
 var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
+var floorf = require( '@stdlib/math/base/special/floorf' );
```

Comment:
```suggestion
var floor = require( '@stdlib/math/base/special/floor' );
```

This change is not desired for the same reasons as discussed elsewhere.